### ISSUE TITLE
broken link

### ISSUE THREAD

**Issue by simonkoehl** [serlo/frontend#215]
on https://de.serlo.org/mathe/zahlen-gr%C3%B6%C3%9Fen/grundrechenarten the link "Aufgaben Rechengesetze und Rechentricks der Addition" is broken

**Comment by simonkoehl** [comment]
may be it has nothing to do with the redesign - but important never the less

**Comment by elbotho** [comment]
broken in serlo.org, returns blank taxonomy.

**Comment by elbotho** [comment]
@yolo-bird could you investigate?

**Comment by yolo-bird** [comment]
@elbotho works in the old design. 
Yet, the topic-folder ist empty. Does this pose any problems in the redesign?

**Comment by simonkoehl** [comment]
@yolo-bird its not related to the redesign but there seams to be a problem never the less: Have a look at the link URL, it is structured different to other taxonomy links

**Comment by elbotho** [comment]
Like with #189 it's confusing for most users.
@yolo-bird would it be okay to not show empty folders?
Or will editors then have a problem?

**Comment by yolo-bird** [comment]
@simonkoehl you're right. Also in the history certain events are missing: https://de.serlo.org/event/history/160961.
To be honest I have no idea, as in the taxonomy it has been created correctly as a topic folder. 
@kulla do you have any ideas here?

**Comment by yolo-bird** [comment]
When you create a curriculum you normale create empty folders first and fill them only afterwards. So it kind of makes sense to be able to create emtpy folders. 

Would be more sensible for authors to have some best practices how to use the description text if an empty folder is created.

**Comment by elbotho** [comment]
@yolo-bird so could we decide that empty folders only display for logged in users?
